app/javascript/javascripts/modules/datepicker.js.coffee

```diff
@@ -52,4 +54,8 @@ class app.Datepicker
     self = this
     $(document).on('click', 'input.date, .control-group .icon-calendar', (e) -> self.show(this))
 
+  onChange: (input, date) ->
```

Comment:
jquery Datepicker does not have a change event by default: https://api.jqueryui.com/datepicker/